packages/database/src/Builder/ModelInspector.php

```diff
@@ -31,8 +31,9 @@ final class ModelInspector
 
     private(set) object|string $instance;
 
-    public function __construct(object|string $model)
-    {
+    public function __construct(
+        private(set) object|string $model,
```

Comment:
Why this change? I don't mind it, but I don't think you do anything with it?

Reply:
I do! https://github.com/tempestphp/tempest-framework/pull/1417/files#diff-c89fc844f3c59f968175abaf32ac8fa8c8cf84fbf3e1eaa2f1fe4d0f4d7d9a58R86